week02/styles/media-query.css

```diff
@@ -0,0 +1,26 @@
+body {
+    font-family: "Roboto", serif;
+
+}
+
+header,
+main,
+footer {
+    max-width: 640px;
+    margin: 1rem auto;
+
+    border: faint black;
```

Comment:
faint black does not exist. Try changing to something else like `black` or `ligthgray`